### PR TITLE
Add temporary indexer to trigger refetching of blocks with missing token transfers

### PR DESCRIPTION
**Pull request by pasqu4le**
Part of #2689

## Checklist for your PR

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [x] I checked whether I should update the docs and did so if necessary
  - [x] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [x] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

File: apps/indexer/lib/indexer/temporary/missing_token_transfers.ex [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    Supervisor.child_spec({BufferedTask, [{__MODULE__, merged_init_opts}, gen_server_options]}, id: __MODULE__)
+  end
+
+  @impl BufferedTask
+  def init(initial, reducer, _) do
+    query =
+      from(
+        s in MissingTokenTransfers.Schema,
+        where: is_nil(s.refetched) or not s.refetched,
```
Comment (ayrat555) [inline apps/indexer/lib/indexer/temporary/missing_token_transfers.ex]: is `is_nil(s.refetched)` the same as  `not s.refetched`?

File: apps/indexer/lib/indexer/temporary/missing_token_transfers.ex [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+    alias Explorer.Chain.Block
+
+    @type t :: %__MODULE__{
+            block_number: Block.block_number(),
+            refetched: boolean() | nil
+          }
+
+    @primary_key false
+    schema "blocks_to_invalidate_missing_tt" do
```
Comment (ayrat555) [inline apps/indexer/lib/indexer/temporary/missing_token_transfers.ex]: How this table will be populated?

**Comment by ayrat555** [comment]
@pasqu4le tests are failing

File: apps/indexer/lib/indexer/temporary/missing_token_transfers.ex (hunk 2 above)
Comment (pasqu4le) [reply apps/indexer/lib/indexer/temporary/missing_token_transfers.ex]: Answered on the other fetcher. These tables are already populated (where there is one)

File: apps/indexer/lib/indexer/temporary/missing_token_transfers.ex (hunk 1 above)
Comment (pasqu4le) [reply apps/indexer/lib/indexer/temporary/missing_token_transfers.ex]: No, `not s.refetched` check if the value is `false` (in case it is not `NULL`).

**Comment by vbaranov** [comment]
Closed in favour of migration to "block_hash" approach

**Pull request closed without merging**